scripts/deploy.ts

```diff
@@ -1,52 +1,42 @@
 // TODO There is a eslint configuration error that needs to be fixed
-/* eslint-disable node/no-missing-import */
-import { namehash, keccak256, toUtf8Bytes, hexZeroPad } from 'ethers/lib/utils'
+import { keccak256, toUtf8Bytes, hexZeroPad } from 'ethers/lib/utils'
 import { ethers } from 'hardhat'
 import { waitForTransactionMined } from './utils'
 
-const ETH_DOMAIN = 'eth'
-const MAIN_SUBDOMAIN = process.env.ENS_DOMAIN || 'fds'
-const FULL_SUBDOMAIN = `${MAIN_SUBDOMAIN}.${ETH_DOMAIN}`
+const DOMAIN = 'fds'
 
 async function deployENS() {
   const ENS = await ethers.getContractFactory('ENSRegistry')
```

Comment:
This contract does not exist... you removed it and started to use the official `@ensdomains` one, didn't you?

Reply:
try to import `ENSRegistry` from that package